tests/test_utils.py

```diff
@@ -164,6 +164,241 @@ def test_list_timeframes(mocker, capsys):
     assert re.search(r"^1d$", captured.out, re.MULTILINE)
 
 
+def test_list_markets(mocker, markets, capsys):
+
+    api_mock = MagicMock()
+    api_mock.markets = markets
+    patch_exchange(mocker, api_mock=api_mock)
+
+    # Test with no --config
+    args = [
+        "list-markets",
+    ]
+    with pytest.raises(OperationalException,
+                       match=r"This command requires a configured exchange.*"):
+        start_list_markets(get_args(args), False)
```

Comment:
This test is fluky and fails if `config.json` exists in the environment.

best have a look at `test_list_timeframes()` on how to mitigate this:

``` python
pargs = get_args(args)
pargs['config'] = None
```